test/tasks/allowTokens.test.ts

```diff
@@ -5,15 +5,15 @@ import fs from 'fs';
 import * as hre from 'hardhat';
 import { ethers } from 'hardhat';
 import sinon from 'sinon';
-import { allowTokens } from '../../tasks/allowTokens';
+import { AllowedTokensArgs, allowTokens } from '../../tasks/allowTokens';
 import { stubReadFileSync } from './utils';
 
 use(chaiAsPromised);
 
 describe('Allow Tokens Script', function () {
```

Comment:
We are missing some testing, ie:

- should allow a list of tokens and verifiers
- should allow a single token and verifiers
- should allow a list of tokens and a single verifier

Reply:
I totally agree with you, and actually, I had also added those tests at some point, but then I had to revert the changes because `sinon-chai` (the library used to spy on some functions) isn't compatible with `smock`, hence I reverted the changes 😓